### ISSUE TITLE
Crash: MiscUtils.dpToPixel

### ISSUE THREAD

**Issue by jonasborggren** [roughike/BottomBar#276]
I get an error on HTC Desire 310 running Android 4.2.2 (not mine, a user)

```
Fatal Exception: java.lang.NoSuchFieldError: android.util.DisplayMetrics.densityDpi
       at com.roughike.bottombar.MiscUtils.dpToPixel(MiscUtils.java:60)
       at com.roughike.bottombar.BottomBar.init(BottomBar.java:877)
       at com.roughike.bottombar.BottomBar.<init>(BottomBar.java:849)
       at com.roughike.bottombar.BottomBar.attach(BottomBar.java:141)
       ....
```

**Comment by roughike** [comment]
Thanks. Will fix ASAP.

**Comment by roughike** [comment]
Should be now fixed in version ```1.3.7```.
